opennms-model/src/main/java/org/opennms/netmgt/model/OnmsMonitoredService.java

```diff
@@ -416,14 +421,20 @@ public void removeMetaData(final String context) {
      * @return a {@link org.opennms.netmgt.model.OnmsIpInterface} object.
      */
     @XmlIDREF
-    @JsonBackReference
+    @JsonIgnore
     @XmlElement(name="ipInterfaceId")
     @ManyToOne(optional=false, fetch=FetchType.LAZY)
     @JoinColumn(name="ipInterfaceId")
     public OnmsIpInterface getIpInterface() {
         return m_ipInterface;
     }
 
+    @JsonProperty("ipInterfaceId")
+    @Transient
+    public Integer getFoo() {
```

Comment:
Fu und Fara oder was?

Should probably be named differently :D

Reply:
Yes, of course.